### ISSUE TITLE
List of functionalities. 

### ISSUE THREAD

**Issue by GabrielvonRoda** [ConnectionScience/TweetPing#9]
Hello Team, 

Tomorrow we should see whether we can come up with a list of functionalities that would be suitable / required for the dashboard for the MIT Connection Science Tweetping Voting app. 

I am herewith starting a list of this once this is complete we can make this a separate issue and attach internal milestones to this and also create said mock-up shots. 

(Dazza it would be good to get your feedback here as we want to make sure your needs are met.) 

The list: 

1.) Tweet Counter 
2.) Colour coding 
3.) Vote outcome pie charts or similar analytics 
4.)
5.)

**Comment by dazzaji** [comment]
Thanks for starting this. I will think it over and circulate for comment this issue link to others working on aspects of this large scale participatory process project.

**Comment by GabrielvonRoda** [comment]
Please see message thread regarding the mock-up ("success criteria") that incorporates the above three functionalities.
